src/transformers/models/bark/configuration_bark.py

```diff
@@ -0,0 +1,330 @@
+    "ylacombe/bark-small": "https://huggingface.co/ylacombe/bark-small/resolve/main/config.json",
+    "ylacombe/bark-large": "https://huggingface.co/ylacombe/bark-large/resolve/main/config.json",
+}
+
+BARK_SUBMODELCONFIG_START_DOCSTRING = """
+    This is the configuration class to store the configuration of a [`{model}`]. It is used to instantiate the model
+    according to the specified arguments, defining the model architecture. Instantiating a configuration with the
+    defaults will yield a similar configuration to that of the Bark
+    [ylacombe/bark-large](https://huggingface.co/ylacombe/bark-large) architecture.
+
+    Configuration objects inherit from [`PretrainedConfig`] and can be used to control the model outputs. Read the
+    documentation from [`PretrainedConfig`] for more information.
+
+    Args:
+        block_size (`int`, *optional*, defaults to 1024):
+        The maximum sequence length that this model might ever be used with. Typically set this to something large
+            just in case (e.g., 512 or 1024 or 2048).
+        input_vocab_size (`int`,, *optional*, defaults to 10_048):
+        Vocabulary size of a Bark sub-model. Defines the number of different tokens that can be represented by the
+            `inputs_ids` passed when calling [`{model}`]. Defaults to 10_048 but should be carefully thought with
+            regards to the chosen sub-model.
+        output_vocab_size (`int`, *optional*, defaults to 10_048):
+        Output vocabulary size of a Bark sub-model. Defines the number of different tokens that can be represented by the:
+            `output_ids` when passing forward a [`{model}`]. Defaults to 10_048 but should be carefully thought with
+            regards to the chosen sub-model.
+        num_layers (`int`, *optional*, defaults to 12):
+            Number of layers.
+        num_heads (`int`, *optional*, defaults to 12):
+            Number of attention heads for each attention layer in the Transformer architecture.
+        hidden_size (`int`, *optional*, defaults to 768):
+            Dimensionality of the "intermediate" (often named feed-forward) layer in the architecture.
+        dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability for all fully connected layers in the embeddings, encoder, and pooler.
+        bias (`bool`, *optional*, defaults to ):
+            True: bias in Linears and LayerNorms, like GPT-2. False: a bit better and faster. Defaults to True.
+        use_cache (`bool`, *optional*, defaults to True):
+            Whether or not the model should return the last key/values attentions (not used by all models).
+        initializer_range (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        n_codes_total (`int`, *optional*, defaults to 8):
+            The total number of [`Encodec`] codebooks. Used in the fine acoustics sub-model.
```

Comment:
Would just say audio codebooks here:
```suggestion
            The total number of audio codebooks predicted. Used in the fine acoustics sub-model.
```